react/11/4/src/selectcolor.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useState } from "react";
+
+export default function SellectColor(props) {
+  const {initColor} = props;
+  const [divStyle, setDivStyle] = useState({ background:initColor ,border: "1px solid black" });
```

Comment:
As the border never changes it should not be in the state, it would be best to keep only the color.
```js
const [color, setColor] = useState("white")
```
And then do this
```js
<div id="colorDiv" style={{ background: color, border: "1px solid black" }}>
```